File: Examples/CSharp/Data/AddOn/NamedRanges/AccessSpecificNamedRange.cs
using System.IO;
using System;
using Aspose.Cells;

namespace Aspose.Cells.Examples.CSharp.Data.AddOn.NamedRanges
{
    public class AccessSpecificNamedRange
    {
        //Source directory
        static string sourceDir = RunExamples.Get_SourceDirectory();

        public static void Run()
        {
            // Opening the Excel file through the file stream
            Workbook workbook = new Workbook(sourceDir + "sampleAccessSpecificNamedRange.xlsx");

            // Getting the specified named range
            Range range = workbook.Worksheets.GetRangeByName("MyRangeTwo");

            if (range != null)
                Console.WriteLine("Named Range : " + range.RefersTo);

            Console.WriteLine("AccessSpecificNamedRange executed successfully.");
        }
    }
}

File: Examples/VisualBasic/Data/AddOn/NamedRanges/AccessSpecificNamedRange.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports System
Imports Aspose.Cells

Namespace Data.AddOn.NamedRanges
    Public Class AccessSpecificNamedRange
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)

            ' Instantiating a Workbook object
            ' Opening the Excel file through the file stream
            Dim workbook As New Workbook(dataDir & "book1.xls")

            ' Getting the specified named range
            Dim range As Range = workbook.Worksheets.GetRangeByName("TestRange")

            If range IsNot Nothing Then
                Console.WriteLine("Named Range : " & range.RefersTo)
            End If
            ' ExEnd:1
        End Sub
    End Class
End Namespace